spring-cloud-huawei-swagger/src/main/java/com/huaweicloud/swagger/SwaggerConfiguration.java

```diff
@@ -26,7 +26,7 @@
  * @Author GuoYl123
  * @Date 2019/12/17
  **/
-@Configuration
+@Configuration(proxyBeanMethods = false)
```

Comment:
I wandor if we don't add this, what's the problem will get? Can you give an example to show this? 

Personanlly, I'd prefer not to add this annotation. There are some reason of personaly preference, but the most important, the target code will conform both to 'Greenwich` and `Hoxton`, so that we do not need to maintain two branches.

Reply:
https://github.com/spring-cloud/spring-cloud-netflix/blob/d91e3bf56dc72ed559e5fa68a7080bef75e40015/spring-cloud-netflix-eureka-server/src/main/java/org/springframework/cloud/netflix/eureka/server/EurekaServerAutoConfiguration.java#L77

https://github.com/alibaba/spring-cloud-alibaba/blob/7b42a9a31e719aa498ee278293fdb02f14618884/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/java/com/alibaba/cloud/nacos/discovery/NacosDiscoveryAutoConfiguration.java#L30

Are you also maintaining two branches now?  if we look from the perspective of "What's the cost of not doing it?" the answer is clearly a zero in the perceivable future

It is itself two incompatible branches, why should we unify the rules,netflix or alibaba Is also maintaining two branches at the same time

Reply:
Thank you for the detail feedback, after carefully anaylse, we will pending this PR to be merged.  It's not the time to make this change.